programs/util/CMakeLists.txt

```diff
@@ -55,17 +55,17 @@ install( TARGETS
    ARCHIVE DESTINATION lib
 )
 
-#add_executable( schema_test schema_test.cpp )
-#target_link_libraries( schema_test
-#                       PRIVATE steem_chain fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
-#
-#install( TARGETS
-#   schema_test
-#
-#   RUNTIME DESTINATION bin
-#   LIBRARY DESTINATION lib
-#   ARCHIVE DESTINATION lib
-#)
+add_executable( schema_test schema_test.cpp )
+target_link_libraries( schema_test
+                       PRIVATE steem_chain fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
+
+install( TARGETS
+   schema_test
```

Comment:
We should add these tests to the Docker file to run as part of CI. Especially because the rc state size code is going to rely on it. We need to make sure this works properly and is not broken by a code change.